src/test/regress/sql/propagate_statistics.sql

```diff
@@ -0,0 +1,43 @@
+CREATE SCHEMA "statistics'test";
+
+SET search_path TO "statistics'test";
+SET citus.next_shard_id TO 980000;
+SET client_min_messages TO WARNING;
+SET citus.shard_count TO 32;
+SET citus.shard_replication_factor TO 1;
+
+-- test create statistics propagation
+CREATE TABLE test_stats (
+    a   int,
+    b   int
+);
+
+SELECT create_distributed_table('test_stats', 'a');
+
+CREATE STATISTICS s1 (dependencies) ON a, b FROM test_stats;
+
+-- test for distributing an already existing statistics
+CREATE TABLE test_stats2 (
+    a   int,
+    b   int
+);
+
+CREATE STATISTICS s2 (dependencies) ON a, b FROM test_stats2;
+
+SELECT create_distributed_table('test_stats2', 'a');
+
+-- test when stats is on a different schema
+CREATE SCHEMA sc1;
+CREATE TABLE tbl (a int, b int);
+SELECT create_distributed_table ('tbl', 'a');
+
+CREATE STATISTICS sc1.st1 ON a, b FROM tbl;
+
+\c - - - :worker_1_port
+SELECT stxname FROM pg_statistic_ext ORDER BY stxname ASC;
```

Comment:
this test easily fails if any other test add statistics and runs before this test. Could we scope this to only show the statistics for the schema's we use in this test suite?

```sql
SELECT stxname
FROM pg_statistic_ext
WHERE stxnamespace IN (
	SELECT oid
	FROM pg_namespace
	WHERE nspname IN ('public')
);
```

with the ordering ofc, and the appropriate namespaces.